config/exporters/monitoring/grafana/base/prometheus-exporter-service.yaml

```diff
@@ -22,74 +25,78 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: exporter-cluster-role-binding
+  name: exporter-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: exporter-role
 subjects:
 - kind: ServiceAccount
-  name: exporter-sa
+  name: kube-rbac-proxy
   namespace: appstudio-grafana-datasource-exporter
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: exporter-cluster-role
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: exporter-service
-  namespace: appstudio-grafana-datasource-exporter
   labels:
-    app: grafana-datasource-exporter
+    app: kube-rbac-proxy
+  name: kube-rbac-proxy
+  namespace: appstudio-grafana-datasource-exporter
 spec:
   ports:
-  - name: http
-    port: 8090
-    targetPort: http
+  - name: https
+    port: 8443
+    targetPort: https
   selector:
-    app: grafana-datasource-exporter
+    app: kube-rbac-proxy
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: exporter-service-deployment
+  name: kube-rbac-proxy
   namespace: appstudio-grafana-datasource-exporter
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: grafana-datasource-exporter
+      app: kube-rbac-proxy
   template:
     metadata:
       labels:
-        app: grafana-datasource-exporter
+        app: kube-rbac-proxy
     spec:
-      serviceAccountName: exporter-sa
+      serviceAccountName: kube-rbac-proxy
       containers:
-      - name: grafana-datasource-exporter
-        image: exporter:latest
+      - name: kube-rbac-proxy
+        image: quay.io/brancz/kube-rbac-proxy:v0.14.0
+        args:
+        - "--secure-listen-address=0.0.0.0:8443"
+        - "--upstream=http://127.0.0.1:8090/"
+        - "--logtostderr=true"
+        - "--v=10"
         ports:
-        - containerPort: 8090
-          name: http
+        - containerPort: 8443
+          name: https
         resources:
           limits:
             cpu: 100m
-            memory: 100Mi
+            memory: 200Mi
           requests:
             cpu: 100m
-            memory: 10Mi
+            memory: 200Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+      - name: exporters
+        image: quay.io/klakshma/myfirstrepo
```

Comment:
update the image to `quay.io/redhat-appstudio/o11y`

Reply:
updated